### PR TITLE
WIP Get all groups

### PR DESCRIPTION
**Pull request by zahodi**

### CODE REVIEW

File: lib/bovine/staticinventory.py [hunk 1]
```diff
@@ -27,12 +27,15 @@ def __init__(self, root_directory=None):
...
+  def debug_print(self):
+    print(self.inventory)
+
   def _get_all_groups(self):
     '''Walk the groups/ dir, saving all found groups
 
     Then, load all variables, hosts and child groups from each group
     '''
-    groups_directory = self.root_directory + '/groups'
+    groups_directory = self.root_directory + 'groups/'
```
Comment (senorsmile) [inline lib/bovine/staticinventory.py]: this may be the fix, although I can't get tox to work on this stupid mac.  test/test2.py works successfully for me.

**Review by senorsmile — CHANGES_REQUESTED** [review]
more changes to come

**Review by zahodi — COMMENTED** [review]
So are you saying that the loop doesn't work or that the merge function doesn't work unless it's python3.5?

**Comment by senorsmile** [comment]
> So are you saying that the loop doesn't work or that the merge function doesn't work unless it's python3.5?

What is this question in response to?

File: test/test_data/static/hosts/main.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+main2:
+  host1:
+    foo2: bar2
```
Comment (senorsmile) [inline test/test_data/static/hosts/main.yml]: the file name must always match the hostname... so this file should be called host1.yml

File: lib/bovine/inventory.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```
Comment (senorsmile) [inline lib/bovine/inventory.py:1]: rename!  sounds good to me

**Review by senorsmile — CHANGES_REQUESTED** [review]
only one change really needed

**Comment by senorsmile** [comment]
get all groups and hosts now working.  need tests.

**Pull request merged**